cobc/codegen.c

```diff
@@ -11751,10 +11751,10 @@ output_internal_function (struct cb_program *prog, cb_tree parameter_list)
 	output_local ("static unsigned int\tinitialized = 0;\n\n");
 	if (prog->flag_recursive) {
 		output_local ("/* Module structure pointer for recursive */\n");
-		output_local ("cob_module\t\t*module = NULL;\n\n");
+		output_local ("cob_module\t\t*module__ = NULL;\n\n");
 	} else {
 		output_local ("/* Module structure pointer */\n");
-		output_local ("static cob_module\t*module = NULL;\n\n");
+		output_local ("static cob_module\t*module__ = NULL;\n\n");
```

Comment:
I'm quite reluctant to change the name here in 3.2 - because that's used in several GDB extensions.
I'm fine with doing that for 4.x though...

Do we _need_ the global name change or is the place you've previously shared (within %s_module_init) enough?
If not, then please change the program name in the testsuite after adding it to cobc.c (cob_csyns) under `P_switch` (we may need to add some more there).